experimental/packages/exporter-logs-otlp-http/src/platform/browser/OTLPLogExporter.ts

```diff
@@ -34,21 +33,13 @@ export class OTLPLogExporter
   implements LogRecordExporter
 {
   constructor(config: OTLPExporterConfigBase = {}) {
-    // load  OTEL_EXPORTER_OTLP_LOGS_TIMEOUT env var
     super(
       {
-        timeoutMillis: getEnv().OTEL_EXPORTER_OTLP_LOGS_TIMEOUT,
```

Comment:
I'm not sure I followed the code path, fully. Does this result in:

- `OTLPExporterBrowserBase()` constructor called with a config without `timeoutMillis` set.
- `OTLPExporterBase()` constgructor called without config.timeoutMillis
- `OTLPExporterBase()` calling `this.timeoutMillis = configureExporterTimeout(config.timeoutMillis);`
- which calls `getExporterTimeoutFromEnv()`
- which attempts to read envvars from `getEnv()` anyway


Perhaps that is fine. This PR is just starting to unwind browser code using `process.env`?

Reply:
Ah, yes I missed that one. But I think keeping that around is fine for now. 
A follow up (a few PRs out, once all the transport code is untangled) will ensure that all of the env var config code is in one place (and used for node.js only).

I adapted the title to reflect that it's not eliminating everything of that yet.